### DISCUSSION TITLE
Introduce Overview HTML page and time-based navigation links

### DISCUSSION THREAD

**Discussion by iher1** [KnugiHK/WhatsApp-Chat-Exporter#96] [Ideas]
Hi,
another idea to make it easier to consume many conversations and very huge chats would be:

- generate an `index.html` page which provides links to the specific chat entry pages
- when working with the `--size` parameter: have some sort of index at the beginning of each html page to quickly jump to
  - a certain subpage, like page 1 2 3 4 5 6 ...
  - jump to the first chat of a specific date, e.g. have links per year: 2020 2021 2022 2023

Regards
iheri1

**Comment by KnugiHK** [comment]
Continue on #101.
